docs/google/best_practices.md

```diff
@@ -4,7 +4,28 @@ This section lists some best practices for creating a circuit that performs well
 on Google hardware devices. This is an area of active research, so users are
 encouraged to try multiple approaches to improve results.
 
-## Use built-in optimizers as a first pass
+This guide is split into three parts:
+*  Getting your circuit to run
+*  Lowering error
+*  Making it run faster
```

Comment:
change the order of these? i.e. 1) get circuits to run 2) make them fast 3) lower error. The thought being: it's easier to try error reduction techniques if it's already faster :) 

otherwise: splitting this doc up into meaningful sections like these is a very good idea

Reply:
Done.